app/models/models_raffle.py

```diff
@@ -0,0 +1,61 @@
+"""Models file for module_tombola"""
+from datetime import datetime
+
+from sqlalchemy import Boolean, Column, DateTime, Float, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from app.database import Base
+from app.models.models_core import CoreUser
+from app.utils.types.raffle_types import RaffleStatusType
+
+
+class Raffle(Base):
+    __tablename__ = "raffle"
+    id: str = Column(String, primary_key=True, index=True, nullable=False)
+    name: str = Column(String, nullable=False)
+    status: RaffleStatusType = Column(
```

Comment:
Use an Enum as column type, check amap_models.Order.collection_slot to see an example.